codeceptConfigHelper.js

```diff
@@ -58,6 +58,7 @@ module.exports = {
     settingsAPI: './tests/pages/api/settingsAPI.js',
     templatesAPI: './tests/ia/pages/api/templatesAPI.js',
     qanAPI: './tests/QAN/api/qanAPI.js',
+    tooltips: './tests/helper/tooltipHelper.js'
```

Comment:
the same here, why can't they just be a page object? or do you think they don't fall in the category of a page object?

Reply:
Moved tooltipHelper contents back into pageObject